bdb/attr.h

```diff
@@ -352,7 +352,7 @@ DEF_ATTR(PLANNER_WARN_ON_DISCREPANCY, planner_warn_on_discrepancy, BOOLEAN, 0)
 DEF_ATTR(PLANNER_EFFORT, planner_effort, QUANTITY, 1)
 DEF_ATTR(SHOW_COST_IN_LONGREQ, show_cost_in_longreq, BOOLEAN, 1)
 /* Delay restarting SC by this many seconds from db up*/
-DEF_ATTR(SC_RESTART_SEC, sc_restart_sec, QUANTITY, 60)
+DEF_ATTR(SC_RESTART_SEC, sc_restart_sec, QUANTITY, 0)
```

Comment:
This was only used by resume_schema_change() for resuming a table alter (in function wait_to_resume()), no?